job-runner/chain/chain.go

```diff
@@ -18,28 +18,39 @@ type chain struct {
 	jcMux    *sync.RWMutex
 	jobChain *proto.JobChain
 
-	runningMux *sync.RWMutex
-	running    map[string]proto.JobStatus // keyed on job id
-	n          uint                       // N jobs ran
+	runningMux         *sync.RWMutex
+	running            map[string]proto.JobStatus // keyed on job id
+	n                  uint                       // N jobs ran
+	sequenceRetryCount map[string]uint            // Number of retries attempted so far
```

Comment:
I do feel like this name is too similar to `job.RetryCount`

`job.RetryCount` - configured number of sequence retries
`sequenceRetryCount` - number of retries attempted so far

so one is static and the other might vary. given this, i don't think i should call them both `RetryCount`

`sequenceRetryNumber` for the running count?

Reply:
There's no `job. RetryCount` afaik?  `sequenceRetryCount` makes sense to me, more than `sequenceRetryNumber`.

Reply:
There is `job.RetryCount`. `RetryCount` field is only set on the job that is the start of the sequence. https://github.com/square/spincycle/blob/c93f793dedec8d682acb2343637bfdba3a20aedd/proto/s2s.go#L24